okcurl/src/main/java/okhttp3/curl/Main.kt

```diff
@@ -205,7 +205,7 @@ class Main : Runnable {
   private fun mediaType(): MediaType? {
     val mimeType = headers?.let {
       for (header in it) {
-        val parts = header.split(":".toRegex()).toTypedArray()
+        val parts = header.split(':')
```

Comment:
This seems like it should have a limit as well

Reply:
https://github.com/square/okhttp/pull/5073